lib/policies/deserializationPolicy.ts

```diff
@@ -3,13 +3,14 @@
 
 import { HttpOperationResponse } from "../httpOperationResponse";
 import { OperationResponse } from "../operationResponse";
-import { OperationSpec } from "../operationSpec";
+import { OperationSpec, isStreamOperation } from "../operationSpec";
 import { RestError } from "../restError";
 import { Mapper, MapperType } from "../serializer";
 import * as utils from "../util/utils";
 import { parseXML } from "../util/xml";
 import { WebResource } from "../webResource";
 import { BaseRequestPolicy, RequestPolicy, RequestPolicyFactory, RequestPolicyOptions } from "./requestPolicy";
+import { isBoolean } from "util";
```

Comment:
This looks like a nodejs built-in which will cause webpack to bring in a polyfill. Would `typeof thing === "boolean"` be adequate?

Reply:
Probably. I thought that was coming from our "../util/utils" file when VS Code recommended it. Thanks for finding that.